packages/fxa-auth-server/lib/senders/emails/templates/cadReminder/index.scss

```diff
@@ -0,0 +1,30 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+@use '../../global.scss';
+
+body > div:first-child {
```

Comment:
I wonder if this selector could be a bit nicer (like with a class instead) or if this'll end up being a global style. I think it's fine to leave for now, just thinking about future templates 🙂